src/Application/Common/Interfaces/IApplicationDbContext.cs

```diff
@@ -8,9 +8,9 @@ public interface IApplicationDbContext
 {
     public DbSet<Department> Departments { get;  }
     public DbSet<User> Users { get; }
-    
     public DbSet<Staff> Staffs { get; }
     public DbSet<Room> Rooms { get; }
+    public DbSet<Locker> Lockers { get; }
```

Comment:
You don't need to throw this here, one more merge from dev should be fine

Reply:
resolved the conflicts